packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -94,11 +103,23 @@ class="fixed inset-0 z-40 flex items-center min-h-screen p-4 overflow-y-auto tra
                     'max-w-5xl' => $width === '5xl',
                     'max-w-6xl' => $width === '6xl',
                     'max-w-7xl' => $width === '7xl',
+                    'fixed inset-4 w-auto' => $isFullscreen,
```

Comment:
Is the `w-auto` really required?

Also, why `inset-4`?

Reply:
'w-auto' isn't required here. 
'inset-4' is to keep the 'padding' around the modal for background clicks. I think is just looks better not going all the way to the edge personally.

Reply:
Fixed in latest commit.